src/account-info/AccountDetailsScreen.js

```diff
@@ -31,7 +30,7 @@ type SelectorProps = $ReadOnly<{|
 |}>;
 
 type Props = $ReadOnly<{|
-  navigation: NavigationScreenProp<{ params: {| email: string |} }>,
+  navigation: NavigationScreenProp<{ params: {| userId: number |} }>,
```

Comment:
nit in commit message: rather than `navigateToAccountDetails` as prefix, I think `account details screen` or `AccountDetailsScreen` is closer to the heart of it. The `navigateToAccountDetails` function is a helper to get to this screen.